sqlalchemy-stubs/sql/crud.pyi

```diff
@@ -1,20 +1,22 @@
 from typing import Any
+from typing import TypeVar
 
-from . import coercions as coercions
-from . import dml as dml
-from . import elements as elements
-from . import roles as roles
-from .. import exc as exc
-from .. import util as util
+from . import elements
+from . import type_api
+from ..util import langhelpers
 
-REQUIRED: Any
+_TE = TypeVar("_TE", bound=type_api.TypeEngine[Any])
 
-class _multiparam_column(elements.ColumnElement):
+REQUIRED: langhelpers._symbol
+
+class _multiparam_column(elements.ColumnElement[_TE]):
```

Comment:
isn't this class private?

Reply:
It seems to be. When the stubs were generated it was retained, so I filled it out.

Reply:
maybe it's used somewhere else, so it was retained because if it?

Reply:
That's what I was thinking, but it doesn't seem to be used anywhere

Reply:
@zzzeek ideas?

Reply:
it's used here:  https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/sql/crud.py#L812

would be used in a statment like this:

```python
>>> from sqlalchemy import table, column
>>> t = table('t', column('a'), column('b'))
>>> stmt = t.insert().values([{"a": 1}, {"a": 2}])

```

looks like the default stringify needs to be opened up for that...

Reply:
I guess we can decide later if we want this or not